### DISCUSSION TITLE
Custom domain for editing

### DISCUSSION THREAD

**Discussion by willwade** [GitbookIO/community#337] [0. General bugs and questions] [answered]
We love gitbook. But the domain/name spins a lot of our non tech staff (the majority) off. We'd like a custom domain for the non published side of things. A lot of our content is internal/reader access only. Something that collects all the resources internally

Or 

Build on the custom domain aspect. Allow spaces to be and viewable and accessible only if you are logged in..

**Comment by aikoii** [comment, marked as answer]
Hey 👋, it looks like the Visitor Authentication visibility option would be the way to go for your needs. That means only people authenticated through your custom backend will be able to check out the documentation in that space. Here is the link for more info: [Visitor Authentication](https://docs.gitbook.com/publishing/visitor-authentication)
